### ISSUE TITLE
File select

### ISSUE THREAD

**Issue by worksofliam** [bendera/vscode-webview-elements#47]
Hi!

We are using the old input box like so:

```html
<vscode-inputbox type="file" id="thefile" name="fileselect"></vscode-inputbox>
```

which allows users to select a file.

I see in the docs that we should be switching to `Textfield`, but it doesn't look like it supports `type=file`.

Any suggestions on what we could do for this?

**Comment by worksofliam** [comment]
I also see that `inputbox` is marked as deprecated, but the examples are still using it. D:

**Comment by bendera** [comment]
Hi,

I fix it ASAP, sorry for that.

**Comment by bendera** [comment]
A new version is bumped.

When I made the new textfield I thought using a file input in a webview is not a real scenario because the extension API has a dedicated function to pick a file from the file system. I was wrong obviously. Now the file picker is ugly but works at least. I will improve its appearance.

The reason I marked the inputbox deprecated is that it is overcomplicated. It is a textfiled and a textarea at the same time. Originally I wanted to emulate the behavior of the source control inputbox with the warning and error messages and the dynamic resizing. But it is unnecessary, everybody just wants a simple textfield or a textarea. And yes, I should update the examples.
